monitoring/mock_uss/geoawareness/parsers/ed269/__init__.py

```diff
@@ -0,0 +1,151 @@
+import json
+from enum import Enum
+from typing import List, Any, Optional, Union, Dict, Tuple
+from implicitdict import ImplicitDict, StringBasedDateTime, StringBasedTimeDelta
+
+
+class Restriction(str, Enum):
+    PROHIBITED = "PROHIBITED"
+    REQ_AUTHORISATION = "REQ_AUTHORISATION"
+    CONDITIONAL = "CONDITIONAL"
+    NO_RESTRICTION = "NO_RESTRICTION"
+
+
+class Reason(str, Enum):
+    AIR_TRAFFIC = "AIR_TRAFFIC"
+    SENSITIVE = "SENSITIVE"
+    PRIVACY = "PRIVACY"
+    POPULATION = "POPULATION"
+    NATURE = "NATURE"
+    NOISE = "NOISE"
+    FOREIGN_TERRITORY = "FOREIGN_TERRITORY"
+    EMERGENCY = "EMERGENCY"
+    OTHER = "OTHER"
+
+
+class YESNO(str, Enum):
+    YES = "YES"
+    NO = "NO"
+
+
+class Purpose(str, Enum):
+    AUTHORIZATION = "AUTHORIZATION"
+    NOTIFICATION = "NOTIFICATION"
+    INFORMATION = "INFORMATION"
+
+
+class UASZoneAuthority(ImplicitDict):
+    name: Optional[str]  # max length: 200
+    service: Optional[str]  # max length: 200
+    email: Optional[str]
+    contactName: Optional[str]  # max length: 200
+    siteURL: Optional[str]
+    phone: Optional[str]  # max length: 200
+    purpose: Optional[Purpose]
+    intervalBefore: Optional[str]
+
+
+class VerticalReferenceType(str, Enum):
+    AGL = "AGL"
+    AMSL = "AMSL"
+
+
+# TODO: Discuss with @ben about Union[PolygonType, CircleType] which throws a not implemented exception
+#  in implicitdict.
+#
+# class PolygonType(ImplicitDict):
+#     type: str = "Polygon"
+#     coordinates: List[ # min 4 items
+#         List[ # 2 items
+#             float
+#         ]
+#     ]
+#
+#
+# class CircleType(ImplicitDict):
+#     type: str = "Circle"
+#     center: List[float] # 2 items
+#     radius: float # > 0
+
+# Start workaround
+class UniversalHorizontalProjectionType(str, Enum):
+    Circle = "Circle"
+    Polygon = "Polygon"
+
+
+class UniversalHorizontalProjection(ImplicitDict):
```

Comment:
nit: I'd probably just name it `CircleOrPolygonType` (but that's a style preference)

Reply:
Done